truffle_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+echo "Adding local binaries in front of PATH"
```

Comment:
Not really needed but for convenience while writing this script.
And if for whatever reasons, node changes the binaries locations, it should still work.